server/package.json

```diff
@@ -12,8 +12,11 @@
 		"url": "https://github.com/amzn/ion-vscode-plugin"
 	},
 	"dependencies": {
-		"vscode-languageserver": "^6.1.1",
-		"vscode-languageserver-textdocument": "^1.0.1"
+		"ion-js": "^3.1.2",
+		"vscode-languageserver": "^5.2.1"
```

Comment:
We went from 6.1.1 to 5.2.1. Why the downgrade?

Reply:
In peter's prototype, it was ^5.2.1. If we use ^6.1.1, we need to make some changes in the prototype. Do we need to do that?

Reply:
It would be better to use the latest major version, 6.x.x

Reply:
Sure.